.github/workflows/ci-syntax-tests.yml

```diff
@@ -0,0 +1,38 @@
+name: CI Syntax Tests
+
+on:
+  push:
+    branches:
+      - 'st4'
+    paths:
+      - '.github/workflows/ci-syntax-tests.yml'
+      - '**.sublime-syntax'
+      - '**/syntax_test_*'
+      - '**.tmPreferences'
+  pull_request:
+    branches:
+      - '**'
```

Comment:
This probably should be `st4` too? We only want to check PRs that are made against the ST4 branch.

That said, I wonder if the CI configuration actually has to live on the main/master branch regardless...

Reply:
I double-checked by merging this PR in my fork with an `st4` branch, and the GH actions seem to run fine!

https://github.com/skyronic/vue-syntax-highlight/tree/st4

The tests are failing but that's not specific to GH actions.

Reply:
I've choosen the wrong package name.

Reply:
If someone creates a PR for a PR, syntax tests should probably run, too. So targeting a dedicated target branch is probably not a good idea. Not sure how likely it is in this repo, but I've done such a couple of times.